up-l1/README.adoc

```diff
@@ -105,8 +105,9 @@ The API signature:
 
 * All UMessage attributes metadata *MUST* be preserved during transmission and available to the receiver
 * *MUST* not manipulate the UMessage payload during transmission
-* *SHOULD* pass resolved UUris (contains both names and numbers)
-* *SHOULD* use the Miro Uri serialization format
+* *MUST* pass UUris which satisfy criteria for xref:../basics/uri.adoc#_micro_uris[Micro Uris]
+* *SHOULD* pass Resolved UUris (contains both names and numbers)
+* *SHOULD* use the Micro Uri serialization format
```

Comment:
The last bullet (from my original description) doesn't make sense if you have the first item.

Reply:
It is not explicit and it comes from my original text :-). and I can't think anymore how to better phrase it so I would just remove the original text please.